addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -10,12 +10,32 @@ import { browser } from "@web/core/browser/browser";
 import { _t } from "@web/core/l10n/translation";
 import { registry } from "@web/core/registry";
 import { debounce } from "@web/core/utils/timing";
+import { loadBundle } from "@web/core/assets";
+import { memoize } from "@web/core/utils/functions";
+
+/**
+ * @return {Promise<{ SfuClient: import("@mail/static/libs/discuss_sfu/discuss_sfu").SfuClient, SFU_CLIENT_STATE: import("@mail/static/libs/discuss_sfu/discuss_sfu").SFU_CLIENT_STATE }>}
+ */
+const loadSfuAssets = memoize(async () => await loadBundle("mail.assets_discuss_sfu"));
 
 const ORDERED_TRANSCEIVER_NAMES = ["audio", "screen", "camera"];
+export const CONNECTION_TYPES = { P2P: "p2p", SERVER: "server" };
 const PEER_NOTIFICATION_WAIT_DELAY = 50;
 const RECOVERY_TIMEOUT = 15_000;
 const RECOVERY_DELAY = 3_000;
-const VIDEO_CONFIG = {
+
+// TODO turn them into call.settings
```

Comment:
todo remaining?

Reply:
removed, related to the poc here: https://github.com/odoo/odoo/pull/140337/